site/docs/v1/tech/tutorials/integrate-ruby-rails-api.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: doc
+title: Integrate Your Ruby on Rails API With FusionAuth
+description: Integrate your Ruby on Rails API with FusionAuth
+navcategory: getting-started
+prerequisites: Ruby, bundler and Rails
+technology: Ruby on Rails
+language: Ruby
+---
+
+## Integrate Your {{page.technology}} API With FusionAuth
+
+{% include docs/integration/_intro-api.md %}
```

Comment:
These links appear to be broken when I try to run this locally. Are we missing some pages?

Reply:
Ooops, I need to commit and push the file up.

Reply:
Should be there now.